pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -110,19 +110,20 @@ type GlobalRoleBinding struct {
 // +kubebuilder:resource:scope=Cluster
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// RoleTemplate holds configuration for a template that is used to create roles and clusterRoles for a cluster or project.
+// RoleTemplate holds configuration for a template that is used to create kubernetes Roles and ClusterRoles
+// (in the rbac.authorization.k8s.io group) for a cluster or project.
```

Comment:
why do we want to add `rbac.authorization.k8s.io group` ?

Reply:
That's the group that the native resources are in. I think it's helpful to be precise here if we are going to refer to "kubernetes" resources.